test/tools/test_save_notes.py

```diff
@@ -0,0 +1,70 @@
+import os
+import sys
+import time
+
+import pytest
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(f"{CURRENT_DIR}/../../tools")
+sys.path.append(f"{CURRENT_DIR}/../../lib")
+
+# pylint: disable=import-error,wrong-import-position
+from save_notes import save_notes
+import utils
+
+
+def test_invalid_string_timestamp():
```

Comment:
I think it is good practice to group all related tests together into a unittest. See https://github.com/green-coding-berlin/green-metrics-tool/blob/dev/test/test_yml_parsing.py

Reply:
I've added parameterization, but am not sure if adding them to a class is needed. I know pytest is ok with both cases, but unsure what is preferred